libs/core-wrappers/src/lib/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { Icon } from '@tacc/core-components';
+import styles from './Navbar.module.css';
+
+export const NavItem: React.FC<
+  React.PropsWithChildren<{ to: string; icon?: string }>
+> = ({ to, icon, children }) => (
+  <NavLink
+    to={to}
+    className={styles['nav-link']}
+    activeClassName={styles['active']}
+    exact={to === '/'}
+  >
+    <div className={styles['nav-content']}>
+      {icon && <Icon name={icon} />}
+      {/* we'll want to set name based on the app */}
+      <span className={styles['nav-text']}>{children}</span>
+    </div>
+  </NavLink>
+);
+
+type NavbarProps = React.PropsWithChildren<unknown>;
+
+const Navbar: React.FC<NavbarProps> = ({ children }) => {
+  return <div className={`${styles['nav-list']}`}>{children}</div>;
+};
+
+export default Navbar;
```

Comment:
It is the same role as Sidebar, however its implementation gets its nav items as children instead of a series of objects that it renders itself. For reference, here is how it is being used in tapis-ui:

https://github.com/tapis-project/tapis-ui/blob/ac65e209cf0d24026eb8a4bd02bb47f11fac4243/src/tapis-app/Apps/_components/AppsNav/AppsNav.tsx

Reply:
Great, thanks. `<NavBar>` usage seems much more React-y.